backend/app/routes/action_logs.py

```
@@ -0,0 +1,58 @@
+from sqlalchemy.orm import Session
+from backend.database.models import ActionLogComment, User
+from backend.database.database import get_db
+from backend.utils.auth import get_current_user
+from typing import List
+
+router = APIRouter(prefix="/action-logs", tags=["action-logs"])
+
+@router.get("/{log_id}/comments")
+async def get_comments(
+    log_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)